features/facebookImport/src/components/activitiesMiniStory/activitiesMiniStory.jsx

```diff
@@ -203,7 +203,7 @@ export const ActivitiesMiniStoryDetails = ({ totalEvents }) => {
                         : selectedYear}
                 </p>
             </div>
-            <InfoButton route="/report/details/activities-info" />
+            <InfoButton infoScreen="activities-info" />
```

Comment:
This means that you're no longer using the concept of "route", or that it's implicit in the `infoScreen`?

Reply:
Yeah, popups don't use route. The point is that popups shouldn't reroute as it makes a lot of stuff easier